Jenkinsfile

```diff
@@ -13,277 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@Library(['private-pipeline-library', 'jenkins-shared']) _
-import com.sonatype.jenkins.pipeline.GitHub
-import com.sonatype.jenkins.pipeline.OsTools
-
-node('ubuntu-zion-legacy') {
-  def commitId, commitDate, version, branch, dockerFileLocations, nexusIqVersion, nexusIqSha
-  def imageId, slimImageId, redHatImageId
-  def organization = 'sonatype',
-      gitHubRepository = 'docker-nexus-iq-server',
-      credentialsId = 'sonaype-ci-github-access-token',
-      imageName = 'sonatype/nexus-iq-server',
-      archiveName = 'docker-nexus-iq-server',
-      iqApplicationId = 'docker-nexus-iq-server',
-      dockerHubRepository = 'nexus-iq-server',
-      tarName = 'docker-nexus-iq-server.tar'
-  GitHub gitHub
-
-  try {
-    if (env.releaseBuild_NAME) {
-      stage('Init IQ Version & Sha') {
-        nexusIqVersion = getVersionFromBuildName(env.releaseBuild_NAME)
-        nexusIqSha = readBuildArtifact(
-          'insight/insight-brain/release',
-          env.releaseBuild_NUMBER,
-          "artifacts/nexus-iq-server-${nexusIqVersion}-bundle.tar.gz.sha256"
-        ).trim()
-      }
-    }
-    stage('Preparation') {
-      deleteDir()
-      OsTools.runSafe(this, "docker system prune -a -f")
-
-      def checkoutDetails = checkout scm
-
-      dockerFileLocations = [
-        "${pwd()}/Dockerfile",
-        "${pwd()}/Dockerfile.slim",
-        "${pwd()}/Dockerfile.rh",
```

Comment:
The original job builds and releases 3 Docker images in the same time (main, slim and RedHat). The new build jobs seem to build and release only the main image (based on `Dockerfile`).

Reply:
@eduard-tita That is correct.  We'll need to create a job for each image, which is a bit of a downside, but the upside is that each job will use the standard pipeline which is well supported.